src/navigations/root-stack.js

```diff
@@ -237,7 +260,7 @@ const AuthenticatedNavigator = () => {
         />
         <AuthenticatedStack.Screen
           name="DiscoveryScreen"
-          component={withSafeAreaView(withKeyboardWrapper(DiscoveryScreenV2))}
+          component={withKeyboardWrapper(DiscoveryScreenV2)}
           options={{
             headerShown: false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `withSafeAreaView` wrapper has been removed from the `DiscoveryScreenV2` component. Ensure that this does not affect the layout of the `DiscoveryScreenV2` on devices with notches or rounded corners (like iPhone X and above).



<!-- This is an auto-generated comment by OSS CodeRabbit -->